CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+# CHANGELOG
+
+## v0.2.0
+
+### Summary
+
+This version introduces [http header matching](https://github.com/aws/aws-app-mesh-roadmap/issues/15) and [explicit priority to routes](https://github.com/aws/aws-app-mesh-roadmap/issues/77), and is a minor version release to reflect the changes both in this release and the previous.
```

Comment:
what about cloudmap?

Reply:
It's not super clear what the benefits are from this description. Is there a bigger change we should be making somewhere else?

Reply:
Cloud Map support was introduced in the previous version and I couldn't think of an eloquent way to say "the previous version should have been a minor version bump".

Reply:
Cloud Map support was introduced in the previous version and I couldn't think of an eloquent way to say "the previous version should have been a minor version bump".